
# Send this whenever it is desired for SMARTS
# to restart the simulation with a new scenario:
string reset_with_scenario_path

AgentSpec[]  initial_agents    # optional (may also be added dynamically via SMARTS/agent_spec topic)
